app/helpers/products_helper.rb

```diff
@@ -34,14 +34,22 @@ def ssh_key
     File.read(target) if target.file?
   end
 
-  def command_btn(title:, key:, display:)
+  # A custom button helper for command line tools
+  #
+  # @param [String] title: The button label
+  # @param [String] key: The key of the command in the CMDS hash ( See: controllers/products_controller.rb )
+  # @param [String] display: The equivalent command to be displayed to the user
+  # @param [optional, String] help: Tooltip text to be provided on hover. ( Default: none )
+  # @param [optional, String] color: The bootstrap color of the button. Ex. "primary", "info", "success", etc. ( Default: "default" )
```

Comment:
These don't follow Yardoc specifications for `@param` (see http://www.rubydoc.info/gems/yard/file/docs/Tags.md#param)